src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -164,13 +164,14 @@ CHIP_ERROR OperationalCredentialsAttrAccess::Read(const ConcreteAttributePath &
 // As per specifications section 11.22.5.1. Constant RESP_MAX
 constexpr uint16_t kMaxRspLen = 900;
 
-void fabricListChanged()
+void fabricListChanged(ListOperation operation)
 {
     emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: Call to fabricListChanged");
 
     // Currently, we only manage FabricsList attribute in endpoint 0, OperationalCredentials cluster is always required to be on
     // EP0.
-    MatterReportingAttributeChangeCallback(0, OperationalCredentials::Id, OperationalCredentials::Attributes::FabricsList::Id);
+    MatterReportingListAttributeChangeCallback(0, OperationalCredentials::Id, OperationalCredentials::Attributes::FabricsList::Id,
```

Comment:
I think we should make this a bit simpler for consumers to use. Instead of having to directly call `MatterReportingListAttributeChangeCallback` (and potentially get some of these args wrong), it'd be better if `AttributeAccessInterface` had a `SetAttributeChanged` method that automatically derived the endpoint and cluster IDs for you, leaving you to just specify the attribute ID.

In the cases where the endpoint ID is missing (i.e the `AttributeAccessInterface` overrides all endpoints), we could do a descriptor walk in that method to find all endpoints that have this cluster instance, and appropriately use that to get the endpoint ID(s).